src/follow_up/components/Intro/index.js

```diff
@@ -0,0 +1,16 @@
+import classes from './styles.module.scss'
+
+function Intro () {
+  return (
+    <div className={classes.messageContainer}>
+      <p>
+        Thanks <b>Jhon</b> for your interest in WeDevelopl!
```

Comment:
- Recibir el nombre por prop para tomarlo de los datos del formulario.
- Dejar un `TODO` donde se pase esa prop para después reemplazarlo por el valor dinámico.
- Typo
```suggestion
        Thanks <b>Jhon</b> for your interest in WeDevelop!
```